<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailGatewayDialog</class>
 <widget class="QDialog" name="EmailGatewayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Email gateway</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Desired email address (including @mailchuck.com)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="radioButtonRegister">
     <property name="text">
      <string>Register on email gateway</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>368</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEditEmailAddress">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>@mailchuck.com</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Email gateway alows you to communicate with email users. Currently, only the Mailchuck email gateway (@mailchuck.com) is available.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QRadioButton" name="radioButtonUnregister">
     <property name="text">
      <string>Unregister from email gateway</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonRegister</tabstop>
  <tabstop>lineEditEmailAddress</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EmailGatewayDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EmailGatewayDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonRegister</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEditEmailAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonUnregister</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEditEmailAddress</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
